### ISSUE TITLE
Notification icon from the web

### ISSUE THREAD

**Issue by MehediH** [CatalystCode/electron-windows-notifications#12]
![image](https://cloud.githubusercontent.com/assets/10274434/21080292/4ba50884-bfa3-11e6-9018-15842cab1b9a.png)

Hi,

I am trying to show an image from the web as a notification icon but it isn't working. It works just fine when I load a local file, however.

Is this a known bug?

Thanks!

**Comment by nadavbar** [comment]
I'm not sure if this is supported by the native UWP APIs. All of the examples I've seen use local resources. Are you sure this is possible for UWP Apps?

**Comment by MehediH** [comment]
Yeah, it is available. Download the "notification visualiser" app from the Windows store, it's from Microsoft and shows all the different notifications that are available which use online images
